docs/releases/1.11.txt

```diff
@@ -157,7 +157,7 @@ Migrations
 Models
 ~~~~~~
 
-* ...
+* Add support for callable defaults in ``get_or_create()`` and ``update_or_create()``
```

Comment:
Add -> Added
add period 
wrap at 79 characters
QuerySet.get_or_create() -- could link similar to this:
```
:meth:`QuerySet.values() <django.db.models.query.QuerySet.values>`
```